src/main/java/org/broadinstitute/hellbender/utils/LoggingUtils.java

```diff
@@ -87,6 +93,22 @@ private static void setJavaUtilLoggingLevel(final Log.LogLevel verbosity) {
         return javaUtilLevelNamespaceMap.get(picardLevel);
     }
 
+    /**
+     * Send log output to file if file name is not null.
+     *
+     * @param fileName  File path for log output
+     */
+    public static void sendLogToFile(final String fileName) {
+        if (fileName != null) {
+            final LoggerContext loggerContext = (LoggerContext) LogManager.getContext(false);
+            final Configuration loggerContextConfig = loggerContext.getConfiguration();
+            final Layout<?> layout = PatternLayout.createLayout(PATTERN_STRING, null, loggerContextConfig,
+                    null, null, false, false, null, null);
+            final FileAppender appender = FileAppender.createAppender(fileName, "", "", fileName, "",
```

Comment:
Do we want append behavior or overwrite?  I would expect overwrite based on the behavior of other gatk arguments, but I'm not sure how the log file is usually used.